### PR TITLE
Fission/dhis2 toolkit update

### PR DESCRIPTION
**Pull request by hemanthReddy06**

### CODE REVIEW

**Comment by biyeun** [comment]
@hemanthReddy06  I fixed the issue with the CTA. Looks like the `<a>` tag wrapped around the `cta` did not work well because the `cta` creates another `<a>` tag and nested anchor tags are not valid HTML

File: dimagi/pages/templates/partials/hubspot_cta.html [hunk 1]
```diff
@@ -5,11 +5,11 @@
       <span>
         {{ fallback_text }}
       </span>
-      <img class="hs-cta-img"
+      <!-- <img class="hs-cta-img"
```
Comment (biyeun) [inline dimagi/pages/templates/partials/hubspot_cta.html:8]: i don't think this should be commented out

**Review by biyeun — CHANGES_REQUESTED** [review]
one small change that needs to be reverted with the commented-out image on the hubspot_cta template

**Comment by hemanthReddy06** [comment]
> one small change that needs to be reverted with the commented-out image on the hubspot_cta template

Updated.

**Review by biyeun — APPROVED** [review]

**Comment by biyeun** [comment]
@hemanthReddy06 thank you! whoa that was quick... 👀  don't forget to enjoy the holiday and sleep too ^_^

**Pull request merged**